hack/errata-check.sh

```diff
@@ -0,0 +1,306 @@
+  print-label "WMCO Version" "v$WMCO_VERSION"
+}
+
+function curl-json {
+  local url=$1
+  local result
+  hostname=$(echo -E "$url" | awk -F/ '{print $3}')
+  # Test connection by only retrieving headers
+  result=$(curl --head --silent --negotiate --user : "$url")
+  exitcode=$?
+  if [ $exitcode -ne 0 ]; then
+    print-error "Unable to curl server. (hostname: $hostname, error: $exitcode)  Connected to VPN?"
+    exit 1
+  fi
+  if echo -E "$result" | grep "401 Unauthorized"; then
+    print-error "401 Unauthorized.  (hostname: $hostname)  Valid kerberos ticket?"
+    exit 1
+  fi
+  # Retrieve the requested content
+  result=$(curl --silent --negotiate --user : "$url")
+  echo -E "$result"
+}
+
+function add-build-tag {
+  # Add a tag for the build nvr (wmco-container-X.Y.Z-n)
+  print-info "Adding build tag:"
+  local build_nvr=$1
+  local wmco_tag
+  wmco_tag="wmco-container-${build_nvr:42}"
+  if git show-ref --tags "refs/tags/$wmco_tag"; then
+    print-green "Tag for $wmco_tag already exists"
+  else
+    print-yellow "Tagging WMCO hash $GITHUB_HASH as $wmco_tag"
+    git tag "$wmco_tag" "$GITHUB_HASH"
+  fi
+}
+
+function get-commits {
+  local start_hash=$1
+  local end_hash=$2
+  mapfile -t MERGE_COMMITS < <(git rev-list --format=oneline --merges "$start_hash...$end_hash")
+}
+
+function get-jira-fix-version {
+  local jira_id=$1
+  local result
+  result=$(curl -s "https://issues.redhat.com/rest/api/2/issue/$jira_id")
+  exitcode=$?
+  if [ $exitcode -ne 0 ]; then
+    print-error "Unable to curl server. (error: $exitcode)  Connected to VPN?"
+    exit 1
+  fi
+  if [[ "$result" == *"Login Required"* ]]; then
+    echo "Restricted"
+    return
+  fi
+
+  fix_version_count=$(echo -E "$result" | jq '.fields.fixVersions | length')
+  if [ "$fix_version_count" -gt 1 ]; then
+    echo "Multiple"
+    return
+  fi
+
+  if [ "$fix_version_count" -eq 0 ]; then
+    echo "Undefined"
+    return
+  fi
+
+  fix_version=$(echo -E "$result" | jq -r '.fields.fixVersions[0].name')
+  echo "$fix_version"
+}
+
+function list-commits {
+  local color=$1
+  local -r format="%-40s %-4s %-65.65s %-13s %-11s %-13s\n"
+  printf "${BLUE}"
+  printf "$format" "Commit Hash" "PR#" "PR Title" "Jira ID" "Fix Version" "Errata Status"
+  printf "%s %s %s %s %s %s\n" "$(printf "=%.0s" {1..40})" "$(printf "=%.0s" {1..4})" "$(printf "=%.0s" {1..65})" "$(printf "=%.0s" {1..13})" "$(printf "=%.0s" {1..11})" "$(printf "=%.0s" {1..13})"
+  printf "$color"
+  for commit in "${MERGE_COMMITS[@]}"
+  do
+    local commit_hash=""
+    local pr_num=""
+    local pr_title=""
+    local jira_id=""
+    local jira_fix_version=""
+    local attached=""
+    commit_hash=$(echo "$commit" | awk '{ print $1 }')
+    pr_num=$(echo "$commit" | awk '{ print substr($5,2) }')
+    pr_title=$(gh pr view "$pr_num" --json title | jq -r .title)
+    jira_id=$(echo "$pr_title" | grep -Eo '(WINC|OCPBUGS)-[0-9]*') || true
```

Comment:
Considering moving the PR title prefixes (`WINC|OCPBUGS`) as a constant at the beginning of the script so that is easier to maintain and reuse.